content/en/quickstart/quickstart-ci.md

```diff
@@ -0,0 +1,195 @@
+---
+type: docs
+category: Quickstart
+description: Integrate Sigstore into your CI system
+title: Sigstore CI Quickstart
+weight: 10
+---
+
+Join us on our [Slack channel](https://sigstore.slack.com/). (Need an [invite](https://links.sigstore.dev/slack-invite)?)
+
+## Sigstore CI quickstart
+
+Sigstore provides two GitHub Actions that make it easy to integrate signing and verifying into your CI system.
+
+- The [`gh-action-sigstore-python` GitHub Action](https://github.com/sigstore/gh-action-sigstore-python) provides the easiest way to generate Sigstore signatures within your CI system. It uses the Sigstore Python language client ([`sigstore-python`](https://github.com/sigstore/sigstore-python)), but can be used to generate Sigstore signatures regardless of your project's language.
+- The [`consign-installer` GitHub Action](https://github.com/marketplace/actions/cosign-installer) installs cosign into your GitHub Action environment, making all features of Cosign available to be used within your CI System.
+
+This quickstart will walk you through the use of the `gh-action-sigstore-python` to [sign](#signing-files-using-your-ci-system) files, which is the quickest way to integrate Sigstore into your CI system. This quickstart also includes a [walkthrough](#using-cosign-within-your-ci-system) of using basic Cosign features in your workflows.
+
+## Using `gh-action-sigstore-python` to sign files within your CI System
+
+This quickstart will show you how to integrate the `gh-action-sigstore-python` GitHub Action into your workflow to generate Sigstore Signatures. The example workflow will sign the file `to_be_signed.txt` in the project's root directory whenever a push is made to the main branch.
+
+Additional information and optional settings can be found in the [project's README](https://github.com/sigstore/gh-action-sigstore-python?tab=readme-ov-file#gh-action-sigstore-python).
+
+### Signing files using your CI system
+
+To following workflow will sign the file `to_be_signed.txt` in the project's root directory whenever a push is made to the main branch. To try it out, make sure to add the file `to_be_signed.txt` to your project, or substitute the file for one in your project.
+
+```console
+name: signing_files
+# This will trigger the workflow to run when commits are pushed to the main branch. This is easy for testing purposes, but for your final workflow use whatever event or schedule makes sense for your project.
+on:
+    push:
+        branches: [ main ]
+jobs:
+  signing_files:
+    runs-on: ubuntu-latest
+    # 'id-token' needs write permission to retrieve the OIDC token, which is required for authentication.
+    permissions:
+      id-token: write
+    steps:
+        # This step ensures that your project is available in the workflow environment.
+        - uses: actions/checkout@v3
```

Comment:
Nitpick: latest is `v4` and we can set `persist-credentials: false` for defense in depth 🙂 

```suggestion
        - uses: actions/checkout@v4
          with:
            persist-credentials: false
```